### DISCUSSION TITLE
oracle 11gr2 line number for sql errors not visible - using default oracle driver

### DISCUSSION THREAD

**Discussion by gthbusr0** [dbeaver/dbeaver#13140] [Q&A] [answered]
System information:
Operating system (distribution) and version: windows 10 enterprise 10.0.18363
DBeaver version: DBeaver 21.1.1
Additional extensions: NONE
Connection specification:
Database name and version : oracle 11gr2
Driver name : default oracle driver bundled with s/w
Do you use tunnels or proxies (SSH, SOCKS, etc)? : no tunnels/proxies

#### Describe the problem you're observing:

hello SMEs,
line numbers for sql errors not visible using Oracle 11gr2 db driver  using default oracle driver. it'll help to display the line numbers that have error for easy troubleshooting. in case I'm missing some config (in addition to what I tried searching in preferences/driver config) appreciate the insights.

oracle sql developer shows something like this:  "error at line: 12 column 44"

#### Steps to reproduce, if exist:

write any syntactically incorrect SQL.

**Comment by p91paul** [comment, marked as answer]
dbeaver usually puts the cursor at the place of the error instead
